docs/app/pages/components/components-sections/input-controls/typeahead/snippets/app.html

```diff
@@ -0,0 +1,27 @@
+<div class="row">
+    <div class="col-md-12 has-feedback">
+        <form class="input-typeahead-form">
+            <input [value]="value"
+                   [(ngModel)]="input"
+                   [ngModelOptions]="{standalone: true}"
+                   [placeholder]="'Enter Text'"
+                   class="form-control"
+                   (keydown.enter)="dropdownOpen = true; $event.preventDefault()"
+                   (click)="dropdownOpen = true; $event.stopPropagation();"
+                   (keydown.escape)="dropdownOpen = false; $event.stopPropagation();"
+                   [attr.aria-controls]="typeahead">
+            <ux-typeahead #typeahead
```

Comment:
Arrow keys should allow navigation of the typeahead items. We have a typeahead key service to handle this, we should use it here.